lib/Core/StatsTracker.cpp

```diff
@@ -176,8 +181,10 @@ StatsTracker::StatsTracker(Executor &_executor, std::string _objectFilename,
                            bool _updateMinDistToUncovered)
   : executor(_executor),
     objectFilename(_objectFilename),
-    statsFile(0),
     istatsFile(0),
+    statsFile(0),
```

Comment:
initialise in header file, not in constructor (unique_ptr?)

Reply:
I tried using unique_ptr and I think it's very ugly as it needs to operate with sqlite3 function. For example you allocate most of the managed objects by passing `sqlite3**` to the allocation object.

Reply:
I checked my version and it's also using raw pointers... Makes sense.